federation/pkg/federation-controller/service/servicecontroller.go

```diff
@@ -790,17 +699,6 @@ func (s *ServiceController) updateFederatedService(fedService *v1.Service, newLB
 		}
 	}
 
-	// Ensure DNS records based on Annotations in federated service for all federated clusters
-	if needUpdate && wantsDNSRecords(fedService) {
```

Comment:
It's not clear to me where this block of code (and the two removed blocks above) have been moved to?